### DISCUSSION TITLE
next-auth + axios + jwt

### DISCUSSION THREAD

**Discussion by sbjang123456** [sbjang123456/TIL#127] [General]
[해당 Discussions 확인](https://github.com/nextauthjs/next-auth/discussions/3550#discussioncomment-1993281)

 *인스턴스(`axios.create`)를 SessionProvider 내에서 사용하기 때문에 사용가능한 것일까?*(**확인필요**)

**Comment by SooJungChae** [comment]
오 이 discussion 은 뭐하는거죠ㅋㅋ
어쩌다가 이 discussion 을 찾게되신거에요?
